sdcm/nemesis.py

```diff
@@ -513,6 +513,25 @@ def disrupt_switch_between_PasswordAuthenticator_and_SaslauthdAuthenticator_and_
         else:
             raise UnsupportedNemesis("SaslauthdAuthenticator can't work without saslauthd environment")
 
+    def disrupt_rolling_config_change_internode_compression(self):
+        def get_internode_compression_new_value_randomly(current_compression):
+            self.log.info(f"Current compression is {current_compression}")
```

Comment:
this looks like a debug log level

Reply:
and better to put it where it originates

Reply:
changed